### PR TITLE
Add urgent-item notice function.

### PR DESCRIPTION
**Pull request by sychangww1121**
Add static/js/globalFunctions.js with non-blocking messagebox
  Add static/html/urgentItem.html to upload urgent-item
  Add routes/urgentItem.js to upload urgent-item to database
  Add static/js/urgentItem.js to handle front-end urgent-item uploading
  modify the highlight effect dealing contained urgent-item's element

### CODE REVIEW

File: routes/checker.js [hunk 1]
```diff
@@ -151,7 +152,8 @@ router.post('/cancelScan', function (req, res) {
           } else {
             res.json({
               state: 'success',
-              order: result['rows']
+              order: result['rows'],
+              updateData: updateData
```
Comment (r05921104) [inline routes/checker.js]: The order and updateData seem not be used by the front end ?

File: static/html/urgentItem.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+          <thead>
+            <tr class="success" style="border-style: none;">
+              <th>物料編號</th>
+              <th>數量</th>
+              <th>上傳狀態</th>
+            </tr>
+          </thead>
+          <tbody id="uploaded_urgentItem"></tbody>
+        </table>
+        <button class="btn btn-success" type="submit" id="submit_button" disabled="disabled">上傳出貨單</button>
```
Comment (r05921104) [inline static/html/urgentItem.html]: It should be "上傳急單"

File: static/js/urgentItem.js [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    let errorMsg = ''
+    // Keep the error message
+    for (let i = 1, row = 0; i < filedataArray.length; i++) {
+      let tr = document.createElement('tr')
+      let rowDataArray = filedataArray[i].split(',')
+      let rowLength = rowDataArray.length
+
+      // Check if the information of one item is abnormal
+      if (rowLength !== 2) {
+        window.alert('每項物料應有2筆資料' + rowLength)
```
Comment (r05921104) [inline static/js/urgentItem.js]: This alert window could be deleted

File: static/js/urgentItem.js [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+      orderInput.setAttribute('style', 'border-style: none;' +
+                                       'font-size: 20px;' +
+                                       'padding: 10px;')
+      orderInput.name = 'Msg'
+      orderInput.value = '格式錯誤，無法上傳'
+      $('#uploaded_urgentItem').append(orderInput)
+    }
+    $('<input>').attr('type', 'hidden')
+                .attr('name', 'total_number')
+                .attr('value', filedataArray.length - 3)
```
Comment (r05921104) [inline static/js/urgentItem.js]: This should be filedataArray.length - 1

File: routes/checker.js (hunk 1 above)
Comment (sychangww1121) [reply routes/checker.js]: Those data are for check whether the cancel scan data is urgent item and highlight the data.
So they'll use for reason.

File: static/html/urgentItem.html (hunk 2 above)
Comment (sychangww1121) [reply static/html/urgentItem.html]: This issue was fixed

File: static/js/urgentItem.js (hunk 3 above)
Comment (sychangww1121) [reply static/js/urgentItem.js]: This issue was fixed

**Pull request merged**